### ISSUE TITLE
ATS is impossible to repair by players

### ISSUE THREAD

**Issue by 0x6273** [space-wizards/space-station-14#27620]
## Description
If ATS is damaged, for example spaced by a meteor event (see screenshot), players have no way to repair it and have to ahelp and hope an admin is on to fix it.

This is due to ATS grid having `ProtectedGridComponent`: https://github.com/space-wizards/space-station-14/blob/05f0fddf5d40a47d5beb0e718afd8c732770307e/Resources/Prototypes/Entities/Stations/base.yml#L57

I'm not sure if there's a good reason why ATS has that comp or if the best fix is to just remove it?

**Reproduction**
Easiest way to repro is to pry a tile and then try to place it again.

**Screenshots**
![image](https://github.com/space-wizards/space-station-14/assets/48867431/0c39fc17-448b-43ac-82cf-ef732a877457)

**Comment by UbaserB** [comment]
this is the same as something like the evac shuttle, it shouldn't be modified. But yeah, it's still a bug

**Comment by deltanedas** [comment]
if someone wants to do a funny like building a bar on the ats, why not let them

thats also impossible with protectedgrid not just repairing it

if the concern is someone making a lattice bridge to the station, ats can just be docked to the station or have a portal made

**Comment by GroggleG** [comment]
> if someone wants to do a funny like building a bar on the ats, why not let them
> 
> thats also impossible with protectedgrid not just repairing it
> 
> if the concern is someone making a lattice bridge to the station, ats can just be docked to the station or have a portal made

agreed, no reason to not other then having it seperate from the station, at that point make it an ftl to prevent sabotage if you really dont want to have people building on it.

**Comment by WarMechanic** [comment]
even with protected grid, its possible to drag shuttle console/parts onto the ATS and have it function like one anyway
